src/main/kotlin/no/nav/syfo/melding/kafka/legeerklaring/KafkaLegeerklaringConsumer.kt

```diff
@@ -31,22 +32,33 @@ class KafkaLegeerklaringConsumer(
                 COUNT_KAFKA_CONSUMER_LEGEERKLARING_READ.increment()
                 val kafkaLegeerklaring = it.value()
                 if (kafkaLegeerklaring != null) {
+                    val arbeidstakerPersonIdent = PersonIdent(kafkaLegeerklaring.personNrPasient)
                     val conversationRef = kafkaLegeerklaring.conversationRef?.refToConversation
                     if (conversationRef != null) {
                         if (
                             connection.hasSendtMeldingForConversationRefAndArbeidstakerIdent(
                                 conversationRef = UUID.fromString(conversationRef),
-                                arbeidstakerPersonIdent = PersonIdent(kafkaLegeerklaring.personNrPasient),
+                                arbeidstakerPersonIdent = arbeidstakerPersonIdent,
                             )
                         ) {
                             connection.createMeldingFraBehandler(
                                 meldingFraBehandler = kafkaLegeerklaring.toMeldingFraBehandler(),
-                                fellesformat = null,
-                                commit = false,
                             )
+                            COUNT_KAFKA_CONSUMER_LEGEERKLARING_STORED.increment()
                         }
                     } else {
-                        // TODO: Handle missing conversationRef
+                        val utgaaende = connection.getUtgaendeMeldingerWithType(
```

Comment:
Kanskje kalle denne noe med forespørsel legeerklaring og mappe til MeldingTilBehandler.
Så kunne man evt lagt logikken her for å sjekke om MeldingFraBehandler hører til MeldingTilBehandler i en av domene-klassene (kanskje noe infix fun 🤓) 
I hvert fall hvis det må sjekkes noe mer enn bare tidspunkt på utgående, f.eks. at innkommende er opprettet/sendt etter utgående.

Reply:
Ja, skal se nærmere på det i en egen PR.